Assets/PatchKit/UnityEditorExtension/Editor/Api/Models/Keys/Error.cs

```diff
@@ -1,14 +1,13 @@
-using Newtonsoft.Json;
+using System;
 
 namespace PatchKit.Api.Models.Keys
 {
-    public struct Error
-    {
-        /// <summary>
-        /// Human-readable error message
-        /// </summary>
-        [JsonProperty("message")]
-        public string Message;
-        
-    }
+[Serializable]
+public struct Error
+{
+    /// <summary>
+    /// Human-readable error message
+    /// </summary>
+    public string message;
```

Comment:
Missing tab indentation in the namespace region.

Reply:
done